sync.sh

```diff
@@ -1,27 +1,24 @@
 #!/bin/bash
 
-# x86_64 or arm64
-[ "$(uname -m)" != "x86_64" ] && USE_ARM64="-arm64"
+# TODO: Change URl and file spec to use edgexfoundry master once developer-scripts PR is merged
+NIGHT_BUILD_URL="https://codeload.github.com/lenny-intel/developer-scripts/zip/multi2"
+FILE_SPEC="developer-scripts-multi2/releases/nightly-build/compose-files/*"
 
-# security or no security
-[ "$SECURITY_SERVICE_NEEDED" != true ] && USE_NO_SECURITY="-no-secty"
+# x86_64 or arm64 for make run option
+[ "$(uname -m)" != "x86_64" ] && USE_ARM64="arm64"
 
-# redis or mongo
-if [ "${DATABASE:=redis}" = redis ]; then
-     PERSIST="-redis"
-else
-     PERSIST="-mongo"
-fi
-
-# nightly or other release
-USE_RELEASE=${RELEASE:-nightly-build}
-if [ "$USE_RELEASE" = "nightly-build" ]; then
-     COMPOSE_FILE="docker-compose-nexus${PERSIST}${USE_NO_SECURITY}${USE_ARM64}.yml"
-     wget -q -O ${COMPOSE_FILE} "https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/nightly-build/compose-files/${COMPOSE_FILE}"
-elif [ "$USE_RELEASE" = "geneva" ]; then
-     COMPOSE_FILE="docker-compose-geneva${PERSIST}${USE_NO_SECURITY}${USE_ARM64}.yml"
-     wget -q -O ${COMPOSE_FILE} "https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/geneva/${COMPOSE_FILE}"
-fi
+# security or no security for make run option
+[ "$SECURITY_SERVICE_NEEDED" != true ] && USE_NO_SECURITY="no-secty"
 
+# Download and extract all the docker compose files
+wget -O compose-files.zip ${NIGHT_BUILD_URL}
+unzip -o -j compose-files.zip ${FILE_SPEC} -d compose-files
 
+(
+  cd compose-files
+  make gen ${USE_NO_SECURITY} ${USE_ARM64}
+  mv docker-compose.yml ../
+)
 
+rm compose-files.zip
+rm -rf compose-files
```

Comment:
Need Makefile for `get-token` and `del-token`

Reply:
For that matter, use `rm -f compose-files.zip` so that it doesn't error if compose-files.zip doesn't exist.  Will save debugging later in case the script is changed to fail on all errors.

Reply:
> Need Makefile for `get-token` and `del-token`

@jinlinGuan , good point. Copied what I did for TAF which doesn't need it after compose is generated. WIll fix.

Reply:
@jinlinGuan , fixed and tested get-token works. Thanks for all your help testing this.

Reply:
There is no Makefile, but some scripts are using it ( bin/setupSecurityserviceTest.sh, bin/cleanSecurityserviceTest.sh, bin/security/setupSecurityAccount.sh ).